src/services/api/roles/roles.resolver.fields.ts

```diff
@@ -20,6 +22,27 @@ export class RolesResolverFields {
     private platformAuthorizationService: PlatformAuthorizationPolicyService
   ) {}
 
+  @UseGuards(GraphqlGuard)
+  @ResolveField('organizations', () => [RolesResultOrganization], {
+    description: 'Details of the roles the contributor has in Organizations',
+  })
+  public async organizations(
+    @Parent() roles: ContributorRoles
+  ): Promise<RolesResultOrganization[]> {
+    return await this.rolesService.getOrganizationRolesForUser(roles.id);
```

Comment:
I don't know who the build passed but getOrganizationRolesForUser accepts roles here.

Reply:
Also fixed in the latest code...

Reply:
correction, it was fixed - looks like something went wrong with a merge...will fix again.